tests/test_package.py

```diff
@@ -277,11 +271,6 @@ def test_torch_install_command_generator():
 
 
 @pytest.mark.gputest
-@pytest.mark.parametrize(
-    "cluster",
-    ["ondemand_cpu_cluster", "v100_gpu_cluster", "k80_gpu_cluster", "a10g_gpu_cluster"],
-    indirect=True,
-)
```

Comment:
I don't think we should be removing these here?

Reply:
added a new on demand fixture for these package tests, issue was we had duplicate `cluster` fixtures which were causing pytest errors

Reply:
One open question is how to incorporate the gpus used here into the default fixtures. @dongreenberg Wdyt about not adding them to the default fixtures, but leaving them as part of the gputest marker where they would only be provisioned if tests are explicitly run with the `gputest` marker?